### ISSUE TITLE
玩家一进去服务器，服务器就崩溃

### ISSUE THREAD

**Issue by assidence** [cxnaive/BlockAction#1]
你好，我正在用 spongeforge-1.12.2-2555-7.1.0-BETA-2842 ，出现了玩家一进去服务器，服务器就崩溃的问题。
[crash-2018-01-21_11.16.25-server.txt](https://github.com/cxnaive/BlockAction/files/1649501/crash-2018-01-21_11.16.25-server.txt)

**Comment by cxnaive** [comment]
新版是7.1.0API...没测试

**Comment by cxnaive** [comment]
7.0.0API是没问题的

**Comment by kencinder** [comment]
I'm having the same problem, plugin was working fine for awhile and now this when trying to connect.

https://pastebin.com/vXDyx60V

**Comment by PaRaDoX833** [comment]
Yeah, me too, it seems like this plugin need to be update in the latest version of sponge API.
Well, i hope will happen...

**Comment by cxnaive** [comment]
Updated.
https://www.mcbbs.net/thread-723050-1-1.html
